activities/TamTamMicro.activity/piano.js

```diff
@@ -119,4 +120,13 @@ enyo.kind({
 	abort: function() {
 		this.endofsound();
 	},
+
+	destroy: function () {
+		var that = this;
+		window.clearInterval(that.job);
+		that.inherited(arguments);
+		document.removeEventListener('keydown', function(event) {
```

Comment:
You should call the removeEventListener with the same function that the one you've registered in addEventListener. Here you pass a function that do the same think but I'm not sure it the same.
So, I guess you must set the function in a variable to be sure to pass the same value.

Reply:
11c503b

Reply:
6dd6371 not sure if using .bind will cause the same problem, but it was broken before and .bind fixed it. But it's not an anonymous function, so maybe this will work?